src/components/feedback/popover/FeedbackPopover.tsx

```diff
@@ -110,7 +110,7 @@ export default function FeedbackPopover({
           </Popover.Button>
           <ReactPortal wrapperId="melodi-feedback-button-portal">
             <Popover.Panel
-              className="melodi-mt-2"
+              className="melodi-mt-2 melodi-z-[9999]"
```

Comment:
Why did you have to bump the z-index? I would have thought that the portal should make this unnecessary, feels a little icky to me

Reply:
To work above the Chakra drawer, which has a z-index of 1500.

Reply:
I thought this is what using portals was supposed to help with, but I just spent too much time reading about stacking contexts and I can't figure out why this doesn't work without the z-index added here.

Funny that the [Chakra docs](https://v2.chakra-ui.com/community/recipes/z-index) say:
> As a general rule, we recommend not exceeding z-index values of more than 1 or 2, and to use stacking contexts to combat more complex stacking issues.

And then they are using 1500 within a chakra portal to render the drawer 🤷 

Anyway, no need for us to spend more time on this right now - I'll will merge/publish this